otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerToDropWizardExportConfiguration.java

```diff
@@ -29,9 +29,16 @@
 @Configuration
 public class OtMicrometerToDropWizardExportConfiguration {
 
-    private static final Logger log = LoggerFactory.getLogger(OtMicrometerToDropWizardExportConfiguration.class);
+    private static final Logger LOG = LoggerFactory.getLogger(OtMicrometerToDropWizardExportConfiguration.class);
+    // Comment here please? I assume these are characters that would be back to export to graphite? I thought - was bad
+    // too or was it  single qutote https://github.com/dropwizard/metrics/issues/1322
+    // There's a GraphiteSanitize class probably should use that too
     private static final Pattern blacklistedChars = Pattern.compile("[{}(),=\\[\\]/]");
 
+    /**
+     * I have no idea what this does. I thought the new micrometer stuff went under new. Thhis appears
+     * to move the DropWizard stuff there
+     */
```

Comment:
It just a plug for DropwizardConfig, I do not think we need it as everything already pre-configured